packages/itinerary-body/src/TransitLegBody/alerts-body.tsx

```diff
@@ -5,12 +5,14 @@ import { Alert } from "@opentripplanner/types";
 import React, { FunctionComponent, ReactElement } from "react";
 import { FormattedMessage } from "react-intl";
 
+import { ExternalLinkAlt } from "@styled-icons/fa-solid";
 import * as S from "../styled";
 import { defaultMessages } from "../util";
 
 const { getUserTimezone, getCurrentDate } = coreUtils.time;
 
 interface Props {
+  agencyName: string;
```

Comment:
`agencyName` is not guaranteed. We should make this optional and have a fallback ("agency")